### PR TITLE
VAS-3

### PR DESCRIPTION
**Pull request by dfernandezramos**

### CODE REVIEW

File: VAS.Core/App.cs [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+		{
+			/* NOTE
+			*  All derived Configs should set the following:
+			*  
+			*  Config.baseDirectory
+			*  Config.configDirectory
+			*  Config.dataDir
+			*  Config.homeDirectory
+			*/
+			Current = appInit;
```
Comment (aoller) [inline VAS.Core/App.cs]: Check Current is not null before initialize.

File: VAS.Core/App.cs [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+						File.Move (configFilepath, App.Current.ConfigFile);
+					} catch (Exception ex) {
+						Log.Exception (ex);
+					}
+				}
+			}
+		}
+
+		// copied from OneplayLongomMatch::CoreServices
+		public static void InitTranslations ()
```
Comment (aoller) [inline VAS.Core/App.cs]: Shouldn't be public

**Comment by dfernandezramos** [comment]
@aoller Ready to review again

File: VAS.Tests/Setup.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
+﻿//
+//  Copyright (C) 2016 dfernandez
```
Comment (ylatuya) [inline VAS.Tests/Setup.cs]: Copyright! Check the rest of files and make sure you configure XS correctly

**Comment by aoller** [comment]
👍

**Pull request merged**